src/agent/index.ts

```diff
@@ -118,4 +138,5 @@ export const sendSiopAuthorizationResponse = agent.sendSiopAuthorizationResponse
 export const keyManagerSign = agent.keyManagerSign
 export const dataStoreGetVerifiableCredential = agent.dataStoreGetVerifiableCredential
 export const dataStoreDeleteVerifiableCredential = agent.dataStoreDeleteVerifiableCredential
+export const createVerifiableCredentialLDLocal = agent.createVerifiableCredentialLDLocal
```

Comment:
I really dislike these aliases and think we should stop using them. It also has problems with autocomplete and your IDE not picking up when methods are renamed etc.